src/compiler/checker.ts

```diff
@@ -674,6 +676,12 @@ namespace ts {
             return result;
         }
 
+        function doesNameResolveToLocalSymbolForExportDefault(exports: SymbolTable, meaning: SymbolFlags, name: string): boolean {

```

Comment:
Actually, nevermind, this just introduces another problem in that we'd actively be working with two symbols for the same entity.